src/js/components/Tabs/Tabs.js

```diff
@@ -377,7 +377,7 @@ const Tabs = forwardRef(
               tabIndex={-1}
               onClick={() => moveByArrowKey('next')}
             >
-              <Box pad={{ vertical: 'xsmall', horizontal: 'small' }}>
+              <Box pad={theme.tabs.header?.nextTabButton?.pad}>
```

Comment:
Soft opinion on naming - since this is already under "tabs" object, do we need to include that in the name? Ideas:
- `tabs.header.next.pad`
- `tabs.header.nextButton.pad`

Reply:
Agree, I like nextButton